packages/dds/tree/src/feature-libraries/sequence-field/sequenceFieldEditor.ts

```diff
@@ -68,53 +76,70 @@ export const sequenceFieldEditor = {
 		return count === 0 ? [] : markAtIndex(index, mark);
 	},
 
-	move(sourceIndex: number, count: number, destIndex: number, id: ChangesetLocalId): Changeset {
+	move(
+		sourceIndex: number,
+		count: number,
+		destIndex: number,
+		detachCellId: ChangesetLocalId,
+		attachCellId: ChangesetLocalId,
+	): Changeset {
 		const moveIn: Mark = {
 			type: "MoveIn",
-			id,
+			id: detachCellId,
 			count,
-			cellId: { localId: id },
+			cellId: { localId: attachCellId },
 		};
 		const moveOut: Mark = {
 			type: "MoveOut",
-			id,
+			id: detachCellId,
 			count,
 		};
 		return moveMarksToMarkList(sourceIndex, count, destIndex, moveOut, moveIn);
 	},
 
-	moveOut(sourceIndex: number, count: number, id: ChangesetLocalId): Changeset {
+	moveOut(sourceIndex: number, count: number, detachCellId: ChangesetLocalId): Changeset {
 		const moveOut: Mark = {
 			type: "MoveOut",
-			id,
+			id: detachCellId,
 			count,
 		};
 		return markAtIndex(sourceIndex, moveOut);
 	},
 
-	moveIn(destIndex: number, count: number, id: ChangesetLocalId): Changeset {
+	moveIn(
+		destIndex: number,
+		count: number,
+		detachCellId: ChangesetLocalId,
```

Comment:
Do you think it makes more sense to call this `moveId` instead of `detachCellId`, since the fact that it's the ID of the detach cell isn't relevant to the move in?

Reply:
Yeah, that's a good point. Thanks for pointing that out.